api/src/main/scala/uk/ac/wellcome/platform/api/controllers/ContextController.scala

```diff
@@ -0,0 +1,21 @@
+package uk.ac.wellcome.platform.api.controllers
+
+import com.twitter.inject.annotations.Flag
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+
+import javax.inject.{Inject, Singleton}
+
+
+@Singleton
+class ContextController @Inject()(
+  @Flag("api.prefix") apiPrefix: String) extends Controller {
```

Comment:
nit: style elsewhere in the project would be to write

```scala
class ContextController @Inject()(
  @Flag("api.prefix") apiPrefix: String
) extends Controller {
```

which I also think is more readable.

Also, I’m slightly surprised we need the `@Flag("api.prefix")` here – maybe we should refactor all this into an `ApiConfig` module that we DI here? It’s a bit ugly. (See the `DynamoConfig` module in the transformer project for an example of what I’m talking about.)